### ISSUE TITLE
AffordanceLLM: Grounding Affordance from Vision Language Models. (arXiv:2401.06341v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#29339]
AffordanceLLM: Grounding Affordance from Vision Language Models. (arXiv:2401.06341v1 [cs.CV])<br>
https://ift.tt/Xt3DFkr<br>
<p>Affordance grounding refers to the task of finding the area of an object with
which one can interact. It is a fundamental but challenging task, as a
successful solution requires the comprehensive understanding of a scene in
multiple aspects including detection, localization, and recognition of objects
with their parts, of geo-spatial configuration/layout of the scene, of 3D
shapes and physics, as well as of the functionality and potential interaction
of the objects and humans. Much of the knowledge is hidden and beyond the image
content with the supervised labels from a limited training set. In this paper,
we make an attempt to improve the generalization capability of the current
affordance grounding by taking the advantage of the rich world, abstract, and
human-object-interaction knowledge from pretrained large-scale vision language
models. Under the AGD20K benchmark, our proposed model demonstrates a
significant performance gain over the competing methods for in-the-wild object
affordance grounding. We further demonstrate it can ground affordance for
objects from random Internet images, even if both objects and actions are
unseen during training. Project site: https://ift.tt/vale1nX
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
